cl/search/models.py

```diff
@@ -1135,8 +1135,51 @@ class Meta:
         proxy = True
 
 
+# @pghistory.track(AfterUpdateOrDeleteSnapshot(), obj_field=None)
```

Comment:
We'll want to remove this comment, right?

Reply:
No, this is a Mixin not a model, if  I understood correctly the doc this decorator should be use in models. I have deleted the comment and I have moved this to model_helpers